info.limpet/META-INF/MANIFEST.MF

```diff
@@ -47,10 +47,12 @@ Require-Bundle: info.limpet.libs;visibility:=reexport,
  org.mwc.cmap.geotools;visibility:=reexport,
  org.eclipse.core.resources,
  org.eclipse.core.runtime,
- org.eclipse.core.filesystem
+ org.eclipse.core.filesystem,
+ javax.xml.bind
 Bundle-Vendor: MWC
-Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-RequiredExecutionEnvironment: JavaSE-11
 Bundle-ClassPath: .,
  libs/xmlpull-1.1.3.1.jar,
  libs/xpp3_min-1.1.4c.jar,
- libs/xstream-1.4.8.jar
+ libs/xstream-1.4.8.jar,
+ libs/jsr-275-1.0-beta-2.jar
```

Comment:
Hmm @G - I think the it's this JSR that got replaced:
https://docs.geotools.org/latest/userguide/welcome/upgrade.html#migrate-to-jsr-363-units

Reply:
Let me check

Reply:
this going to be fun im on it